src/core/app.lua

```diff
@@ -99,7 +99,7 @@ end
 
 -- Run app:methodname() in protected mode (pcall). If it throws an
 -- error app will be marked as dead and restarted eventually.
-local function with_restart (app, method)
+function with_restart (app, method)
```

Comment:
@eugeneia Yep, I made `with_restart` global was it was defined after the ingress drop monitor code. OK, I can move that code after the definition of `with_restart`.